app/ChartHistory.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import moment from 'moment';
+import DatePicker from './DateTimePicker/DatePicker.jsx';
+import TimePicker from './DateTimePicker/TimePicker.jsx';
+import './chart-history.scss';
+
+class ChartHistory extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            date: moment().format('YYYY/MM/DD'),
+            time: '00:00',
+        };
+    }
+    onChange({ target }) {
+        if (target.name === 'date') {
+            this.setState({
+                date: target.value,
+            });
+        }
+
+        if (target.name === 'time') {
+            this.setState({
+                time: target.value,
+            });
+        }
+
+        this.props.onChange(`${this.state.date} ${this.state.time}`);
+    }
+    render() {
+        return (
+            <div className="ciq-chart-history">
+                <strong>Historical Data:</strong>
```

Comment:
this needs to be translated as well. I think for the time being we can just bundle the app and library translations together (there are not many strings in the app anyway)